core/src/inputs/advice.rs

```diff
@@ -15,7 +7,7 @@ use sparse_merkle_tree::SparseMerkleTree;
 #[derive(Clone, Debug)]
 pub enum AdviceSet {
     MerkleTree(MerkleTree),
-    SparseMerkleTree(SparseMerkleTree),
+    SparseMerkleTree(SimpleSmt),
     MerklePathSet(MerklePathSet),
 }
```

Comment:
Should this move to `processor` as well? Or we need to keep it in `core` for some reason?

Reply:
Seems it's a leftover of the previous changes. Moved to `processor` as `MerkleSet`!